test/opt/def_use_test.cpp

```diff
@@ -1318,7 +1318,9 @@ TEST_P(KillInstTest, Case) {
       BuildModule(SPV_ENV_UNIVERSAL_1_1, nullptr, tc.before);
   ASSERT_NE(nullptr, module);
   ir::IRContext context(std::move(module), spvtools::MessageConsumer());
-  context.BuildDefUseManager();
+
+  // Force a re-build of def-use manager the next time it's requested.
+  context.InvalidateAnalyses(ir::IRContext::Analysis::kAnalysisDefUse);
 
```

Comment:
this is not testing that KillInst is updating the def-use manger anymore.  We are suppose to build it and have it valid for the call to KillInst.  Then we check the chains at the end.  Now it is building it at the end.

Reply:
Ah, yeah.  If I add a call to get_def_use_mgr(), I would restore the original semantics of the test, right?